config/beebop.yml

```diff
@@ -0,0 +1,29 @@
+docker:
+  network: beebop_nw
+  prefix: beebop
+proxy:
+  host: localhost
+  port_http: 80
+  port_https: 443
+  image:
+    repo: mrcide
+    name: beebop-proxy
+    tag: bacpop-65
+server:
+  image:
+    repo: mrcide
+    name: beebop-server
+    tag: bacpop-65
+  port: 4000
+api:
+  image:
+    repo: mrcide
+    name: beebop-py
+    tag: main
+  db_location: ./storage/GPS_v4_references
```

Comment:
For prod do we want to use the big DB Marie? Do you know if this works Alex?

Reply:
Ah yeah, I haven't tried with the full fb but should change this config for prod and give it a try

Reply:
Have updated to make this configurable now